eta/core/datasets.py

```diff
@@ -662,7 +662,7 @@ def split(self, split_fractions=None, descriptions=None,
         return dataset_list
 
     def add_file(self, data_path, labels_path, move_files=False,
-                 error_on_duplicates=False):
+                 error_on_duplicates=False, link_data=False):
```

Comment:
I think using `mode` or `data_method` argument with accepted values being (`copy`, `move`, `link`, `symlink`) would be better than adding all the options as exclusive or flags, i.e. `link_data` or `symlink_data`.

Reply:
Yes, I like this idea.